src/components/KudosList.js

```diff
@@ -19,6 +19,7 @@ module.exports = React.createClass({
 
 			return (
 				<KudosNote author={author.fullname}
+					username={author.username}
```

Comment:
Let's pass in the entire `author` object in the `author` prop - it seems more intuitive from an API point of view. Plus, we're using more than one of its properties so it makes sense.